### PR TITLE
Update list of maintainers, add section for past maintainers

### PR DESCRIPTION
**Pull request by ckamm**
* Update list of maintainers
* Fix link to binary downloads

@jturcotte @dragotin @danimo @cryptomilk Let me know if you don't want to appear like this, and if you'd like your email addresses updated!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,6 +44,24 @@ If you want to contact us, e.g. before starting a more complex feature,
...
+* Olivier Goffart <ogoffart@owncloud.com>
+* Christian Kamm <mail@ckamm.de>
+
+ownCloud Desktop Client is developed by the ownCloud community and receives
+patches from a variety of authors.
+
+Past maintainers:
+
+* Klaas Freitag <freitag@owncloud.com>
+* Daniel Molkentin <danimo@owncloud.com>
```
Comment (danimo) [inline README.md]: daniel@molkentin.de, please

**Comment by danimo** [comment]
Have you also changed the maintainer notes in the about dialog already? I keep receiving pleas for help that I cannot respond to.

**Comment by ckamm** [comment]
@danimo No, very good point!

File: README.md [hunk 2]
```diff
@@ -56,6 +44,24 @@ If you want to contact us, e.g. before starting a more complex feature,
...
+* Jocelyn Turcotte <jturcotte@woboq.com>
```
Comment (jturcotte) [inline README.md]: That address isn't valid anymore. You can just remove me from the list, the git log is paying tribute enough for my need.

File: README.md [hunk 3]
```diff
@@ -56,6 +44,24 @@ If you want to contact us, e.g. before starting a more complex feature,
...
+* Andreas Schneider <asn@cryptomilk.org>
+
```
Comment (guruz) [inline README.md]: Add a link to https://github.com/owncloud/client/graphs/contributors ? The section title has "Contributors"...

**Comment by guruz** [comment]
> Have you also changed the maintainer notes in the about dialog already? 

We'll introduce an 'About' menu entry in the tray dialog, FYI. To make the general settings less high.

File: README.md (hunk 3 above)
Comment (ckamm) [reply README.md]: That link makes a lot of sense - it would even work instead of the explicit list. Would you agree to just have the link?

Comment (guruz) [reply README.md]: I don't mind very much but i'd prefer to have both link and names.

**Comment by ckamm** [comment]
Updated. I'll not touch the "About" theme string for now since @guruz is already reworking that.

**Pull request merged**